documentation/README.md

```diff
@@ -0,0 +1,267 @@
+# Percona Monitoring and Management (PMM) Documentation
+[![render](https://img.shields.io/badge/pmm-render-Green)](https://pmm-doc.onrender.com/)
+[![Build](https://github.com/percona/pmm/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/percona/pmm/actions/workflows/documentation.yml)
+[![Helm](https://github.com/percona/pmm/actions/workflows/helm-tests.yml/badge.svg?branch=main)](https://github.com/percona/pmm/actions/workflows/helm-tests.yml)
+[![Podman](https://github.com/percona/pmm/actions/workflows/podman-tests.yml/badge.svg?branch=main)](https://github.com/percona/pmm/actions/workflows/podman-tests.yml)
+
+[Percona Monitoring and Management] (PMM) is a database monitoring solution that is free and open-source.
+
+This repo holds the source files for the official [PMM technical documentation].
+
+To contribute to that documentation, you can:
+
+- **report a general problem** -- open a [Jira] issue.
+
+- **fix a problem yourself** -- Use the *Edit this page* link to take you to the Markdown source file for that page. Make your changes (you'll have to fork the repo unless you're Percona staff) and submit a PR which we'll review and adjust where necessary before merging and publishing. If the changes are more than a few lines, you might want to build the website locally to see how it looks in context. That's what the rest of this README covers.
+
+## Introduction
+
+We use [MkDocs] to convert [Markdown] files into a static HTML website (or [PDF](#pdf)). This process is called [*building the documentation*](#building-the-documentation).
+
+The documentation source files are in the `docs` directory. (Other files in this repo are explained in [Directories and files](#directories-and-files).)
+
+The two major PMM versions are kept in separate branches:
```

Comment:
should we menion v3 here as well?

Reply:
Sure, done.

I just had to remove v1 for now, since it's not there yet.